insights/parsers/engine_config.py

```diff
@@ -0,0 +1,83 @@
+from collections import namedtuple
```

Comment:
Docstring file header is necessary, please add it (see example in other parses  e.g. "tomcat_xml.py")